### PR TITLE
update naming conventions

### PR DESCRIPTION
**Pull request by manikantakailasa**
must replace values

site.url to siteUrl
site.title to siteTitle
admin_url to adminUrl
base_url to homeUrl
remove the `sdk` property and let its contents live at the top level

Jira - https://jira.newfold.com/browse/PRESS0-1194

### CODE REVIEW

File: src/components/WonderCart.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ export function WonderCart(props) {
         cardLink={
           (canAccessGlobalCTB &&
             !hasYithExtended &&
-            NewfoldRuntime.sdk.ecommerce.brand_settings.wondercartBuyNow) || ''
+            NewfoldRuntime.ecommerce.brand_settings.wondercartBuyNow) || ''
```
Comment (ramyakrishnai) [inline src/components/WonderCart.js:97]: In NewfoldRuntime, we're not getting ecommerce value currently, can you make those changes as well ?

**Comment by ramyakrishnai** [comment]
use	add_filter( 'newfold_runtime', array( $this, 'add_to_runtime' ) ); in ecommerce.php

File: src/components/WonderCart.js (hunk 1 above)
Comment (manikantakailasa) [reply src/components/WonderCart.js:97]: done

**Review by ramyakrishnai — DISMISSED** [review]

**Comment by aratidgr8** [comment]
@manikantakailasa  Is this change QA tested for all dependent `newfold-labs` modules, without this confirmation we should not merge this branch

**Review by ramyakrishnai — APPROVED** [review]

**Pull request merged**